zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -657,6 +657,9 @@ class CephRGWTest(test_utils.BaseCharmTest):
     primary_rgw_unit = 'ceph-radosgw/0'
     secondary_rgw_app = 'secondary-ceph-radosgw'
     secondary_rgw_unit = 'secondary-ceph-radosgw/0'
+    cloud_sync_rgw_app = 'cloud-sync-ceph-radosgw'
+    cloud_sync_default_s3_app = 's3-default'
+    cloud_sync_dev_s3_app = 's3-dev'
```

Comment:
Need some comments around why two S3 apps are required for this test.

Reply:
I will add more comments here.

Reply:
Added comments to clarify the purpose of these S3 apps in the Zaza cloud-sync tests bundle.

See the added comments [here](https://github.com/openstack-charmers/zaza-openstack-tests/compare/4582566842a449dd0204df9af5e39fe02fb3ef79..4c2fea973bd94ba223df21ba396235cb599b38a4).